File: web2-Project/src/app/entities/car/car.ts
export class Car {
    id: number;
    image: string;
    brand: string;
    model: string;
    type: string;
    cm3: number; 
    kw: number; 
    doors: number;
    gear: string;
    fuel: string;
    year: number;
    pricePerDay: number;
    seats: number;
    freeSeats: number;
    trunk: number;
    raiting: Array<number>;
    averageRating: number;

    babySeat: boolean;
    navigation: boolean;
    roofRack: boolean;
    

    constructor(id: number,
                image: string, 
                brand: string, 
                model: string, 
                type: string, 
                cm3: number,
                kw: number,
                doors: number,
                gear: string,
                fuel: string,
                year: number, 
                seats: number, 
                freeSeats: number, 
                trunk: number, 
                pricePerDay: number,
                raiting: Array<number>,
                average: number,
                babySeat: boolean,
                navigation: boolean,
                roofRack: boolean) {
        this.id = id;
        this.image = image;
        this.brand = brand;
        this.model = model;
        this.type = type;
        this.gear = gear;
        this.cm3 = cm3;
        this.kw = kw;
        this.doors = doors;
        this.fuel = fuel;
        this.year = year;
        this.seats = seats;
        this.freeSeats = freeSeats;
        this.trunk = trunk;
        this.pricePerDay = pricePerDay;
        this.raiting = new Array<number>();
        this.raiting = raiting;
        this.averageRating = average;
        this.babySeat = babySeat;
        this.navigation = navigation;
        this.roofRack = roofRack;
    }
}

File: MAANPP20/MAANPP20/Common/Models/Cars/Car.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Common.Models.Cars
{
	public enum BrandCarEnum
	{
		AlfaRomeo = 1,
		Audi,
		BMW,
		Citroen,
		Fiat,
		Ford,
		Mercedes,
		Peugeot,
		Porsche,
		Toyota,
		Volkswagen,
		Volvo,
		Yugo,
		Skoda
	}
	public enum TypeCarEnum
	{
		Convertable = 1,
		Crossoover,
		Hatchback,
		Luxuary,
		Minivan,
		Sedan
	}
	public enum GearCarEnum
	{
		Manual = 1,
		Automatic
	}
	public enum DoorsCarEnum
	{
		Two = 1,
		Four
	}
	public enum SeatsCarEnum
	{
		Two = 1,
		Four,
		Five,
		Six,
		Seven,
	}
	public enum FuelCarEnum
	{
		Persol = 1,
		Diesel,
		Electric,
		Hybrid
	}
	public class Car
	{
		[Key]
		public int idCar { get; set; }

		public int idService { get; set; }
		public BrandCarEnum Brand { get; set; }
		public string Model { get; set; }
		public TypeCarEnum Type { get; set; }
		public GearCarEnum Gear { get; set; }
		public FuelCarEnum Fuel { get; set; }
		public DateTime Year { get; set; }
		public int Cm3 { get; set; }
		public int Kw { get; set; }
		public DoorsCarEnum Doors { get; set; }
		public SeatsCarEnum Seats { get; set; }
		public int FreeSeats { get; set; }
		public int Trunk { get; set; }
		public ICollection<Ocena> OceneAuta { get; set; }
		public ICollection<RezervacijaOdDo> RezervacijeAutaOdDo { get; set; }
		public double PricePerDay { get; set; }
		public string CarImage { get; set; }
		public bool BabySeat { get; set; }
		public bool Navigation { get; set; }
		public bool RoofRack { get; set; }

		public bool deleted { get; set; }


	}
}
